src/styles/modules/post/post.scss

```diff
@@ -107,3 +107,8 @@
     border-width: 1px 0 1px 0 !important;
   }
 }
+
+.grid-row > .Comments > .CommentsList {
```

Comment:
Try to avoid this pattern as much as possible. In this case if someone change `.grid-view` in the future it will break the style in other parts and is kinda hard to find and debug.